### PR TITLE
Examples: Remove official-storybook

### PR DESCRIPTION
**Pull request by ndelangen**
remove examples/official-storybook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,17 +89,18 @@ For additional help, join us in the [Storybook Discord](https://discord.gg/story
...
-| [Android, iOS, Flutter](https://github.com/storybookjs/native) | [![Storybook demo](https://img.shields.io/npm/v/@storybook/native/latest?style=flat-square&color=blue&label)](https://storybookjs.github.io/native/@storybook/native-flutter-example/index.html) | [![Native](https://img.shields.io/npm/dm/@storybook/native?style=flat-square&color=eee)](https://github.com/storybookjs/native)                   |
+| ?                                                              | TODO, the link below will break                                                                                                                                                                  | [![React](https://img.shields.io/npm/dm/@storybook/react?style=flat-square&color=eee)](code/renderers/react)                                       |
```
Comment (ndelangen) [inline README.md:92]: @storybookjs/documentation @jonniebigodes We have a bunch of places where we demo some of our examples. The official example is obviously one of those.

This PR removes that example. Meaning these links be become dead-ends.

We'll have to come up with something to serve this purpose...

Comment (ndelangen) [reply README.md:92]: @tmeasday Michael mentioned to me that this documentation change was on your plate?

Comment (tmeasday) [reply README.md:92]: @shilman @ndelangen I was only planning on updating the contributing docs. I think this job of "update all documentation to remove reference to uploaded examples and replace with references to uploaded sandboxes" is probably a whole task of it's own!

**Comment by JReinhold** [comment]
I pushed a fix for the failing smoketest of `svelte-kitchen-sink`.

- I don't know why it started failing now
- I don't know why warnings would cause it to fail completely

We're removing the example soon so I just wanted to do the minimal fix, which was to make Svelte ignore the a11y warnings.

**Comment by shilman** [comment]
Followups:
- Need a storybook for developing Storybook's UI
- Need demo storybooks to link to throughout our docs
- Need (?) a demo storybook for interaction testing (maybe -- see AccountForm)

**Review by shilman — APPROVED** [review]
YOLO!!! Let's goooooo!!!! 🚀

**Pull request merged**